.github/workflows/ci-e2e-pr.yaml

```
@@ -0,0 +1,51 @@
+      - uses: actions/checkout@11d5960a326750d5838078e36cf38b85af677262 # ratchet:actions/checkout@v4
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@f75dd6564bb646f95277dc8c3b80612e46a4a1ea # ratchet:fkirc/skip-duplicate-actions@v3.4.1
+        with:
+          cancel_others: false
+          paths_ignore: '["**/*.md", "**/*.adoc", "LICENSE"]'
+          do_not_skip: '["pull_request", "push", "merge_group", "workflow_dispatch", "schedule"]'
+  e2e-pr:
+    name: E2E PR Tests
+    needs: pre-job
+    if: needs.pre-job.outputs.should_skip_e2e != 'true'
+    uses: ./.github/workflows/ci-e2e-0.yaml
```

actionlint (1.7.12)

[error] error while parsing reusable workflow "./.github/workflows/ci-e2e-0.yaml": yaml: unmarshal errors:   line 98: yaml: on.workflow_call.secrets must be mapping node but scalar node was found at line:98, col:14

(workflow-call)